Resources/Prototypes/Catalog/VendingMachines/Inventories/donut.yml

```
@@ -0,0 +1,9 @@
+- type: vendingMachineInventory
+  id: DonutInventory
+  name: Donut
+  spriteName: donut
+  startingInventory:
+    FoodDonutChocolate: 5
+    FoodDonutApple: 5
+    FoodDonutPink: 5
+    FoodDonutBungo: 5
```

should look something like this for inventory imo, chocolate are the most common

```suggestion
    FoodDonutChocolate: 5
    FoodDonutApple: 3
    FoodDonutPink: 3
    FoodDonutBungo: 3
```